Gruntfile.js

```diff
@@ -424,14 +431,16 @@ module.exports = (grunt) => {
             // Open-sourced files or new files that are missing a license
             else {
 
+                let squashedFileContent = squash(fileContent);
+
                 // File has commercial license but is not added to the list of commercial files
-                if (fileContent.includes(commercialLic)) {
+                if (squashedFileContent.includes(squash(commercialLic))) {
```

Comment:
Ignores white space differences.
WebStorm lines up the asterisks whenever I pasted in a multi-line comment.
I realized licenses were never meant to be pasted in manually, so we could do without this.